tests/robot-tests/run_tests.py

```diff
@@ -152,18 +152,32 @@
 else:
     load_dotenv('.env.' + args.env)
 
-assert os.getenv('TIMEOUT') is not None
-assert os.getenv('IMPLICIT_WAIT') is not None
-assert os.getenv('PUBLIC_URL') is not None
-assert os.getenv('ADMIN_URL') is not None
-assert os.getenv('PUBLIC_AUTH_USER') is not None
-assert os.getenv('PUBLIC_AUTH_PASSWORD') is not None
-assert os.getenv('RELEASE_COMPLETE_WAIT') is not None
-assert os.getenv('WAIT_MEDIUM') is not None
-assert os.getenv('WAIT_LONG') is not None
-assert os.getenv('WAIT_SMALL') is not None
-assert os.getenv('FAIL_TEST_SUITES_FAST') is not None
-assert os.getenv('IDENTITY_PROVIDER') is not None
+
+assert os.getenv('TIMEOUT') is not None, 'Missing environment variable: TIMEOUT. refer to `.env.example` for more information.'
```

Comment:
All the extra messages seem a bit overkill. If you feel like it's actually helpful, then would suggest that we refactor this into a function so that we're not copy and pasting the same message over and over again.

Reply:
Yeah on second thoughts I think the out of the box AssertionError message is good enough i.e.:

```
  File "/Users/lukehive/srv/dev/explore-education-statistics/tests/robot-tests/run_tests.py", line 186, in <module>
    assert os.getenv('WAIT_MEMORY_CACHE_EXPIRY')
AssertionError
```